GetIndexNames/index.js

```diff
@@ -2,9 +2,11 @@ const azureApimRequest = require('../lib/AzureApimRequest');
 const utilities = require('../lib/utilities');
 
 module.exports = async function getIndexNames(context, apimApiName) {
+  context.log({ apimApiName });
   const response = await azureApimRequest(apimApiName);
 
   const activeServiceUrl = response.properties.serviceUrl;
+  context.log({ activeServiceUrl });
 
```

Comment:
Could some additional information be added to increase the usefulness of this message within the logs please?